builtin/credential/jwt/path_config.go

```diff
@@ -431,7 +431,9 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 	if err != nil {
 		return nil, err
 	}
-	if err := req.Storage.Put(ctx, entry); err != nil {
+	if err := logical.WithTransaction(ctx, req.Storage, func(storage logical.Storage) error {
```

Comment:
Don't we want the transaction created earlier on the read call as well, for consistency between when the read occurred and when the write executed?

Reply:
yes that's a good point -- i wasn't sure if the cost of a longer transaction would be too high